en/docs/get-started/about-this-release.md

```diff
@@ -12,227 +12,58 @@ For more information on WSO2 API Manager, see the [overview]({{base_path}}/getti
-
-    Ability to configure Rate Limiting for an API Gateway Cluster via the Redis server so that the request counters maintained for burst control and backend rate limiting will be replicated across the API Gateway cluster when working with multiple API Gateway nodes.
-
-    **[Learn more]({{base_path}}/design/rate-limiting/advanced-topics/configuring-rate-limiting-api-gateway-cluster/)**
-
-??? note "Tenant configurations migration from the registry to the database"
-
-    You can now perform more advanced configurations for tenant configuration migration from the registry to the database.
-
-    **[Learn more]({{base_path}}/administer/advanced-configurations/)**
-
-??? note "File Inbound Endpoint subdirectory processing capability"
-
-    VFS Inbound Endpoints are capable of handling files inside subdirectories of the specified URL. This enables users to process files recursively inside a directory.
-
-    **[Learn more]({{base_path}}/reference/synapse-properties/inbound-endpoints/polling-inbound-endpoints/file-inbound-endpoint-properties/)**
-
-??? note "Improve Service Catalog to work with Hot Deployments"
-
-    You can now publish services to Service Catalog for hot deployments and include metadata files of the selected artifacts.
-
-    **[Learn more]({{base_path}}/tutorials/integration-tutorials/service-catalog-tutorial/)**
-
-??? note "Support to change lifecycle state of API Products using apictl"
-
-    You can now change the status of an API or API Product in an environment using apictl.
-
-    **[Learn more]({{base_path}}/install-and-setup/setup/api-controller/managing-apis-api-products/managing-apis-and-api-products/#change-status-of-an-api-or-api-product-in-an-environment)**
-
-??? note "Exposing proxy services via the Service Catalog"
-
-    This feature generates metadata artifacts for the APIs and Proxy services that have been created inside older Integration Studio workspaces.
-
-    **[Learn more]({{base_path}}/integrate/develop/generate-service-catalog-metadata/)**
-
-??? note "Support for SMB and WebDAV protocols"
-
-    This feature adds support to read from or write to files opened via SMB and WebDAV protocols.
-
-    **[Learn more](https://siddhi-io.github.io/siddhi-io-file/)**
-
-??? note "Support Protobuf map-type for Kafka"
-
-    This improvement adds support to decode Protobuf messages via a schema registry with Kafka.
-
-    **[Learn more](https://siddhi-io.github.io/siddhi-io-kafka/api/latest/)**
-
-## Removed features and functionalities
-
-These features are unsupported and removed from WSO2 API Manager 4.1.0 onwards.
-
-- **DAS Message Tracer**
-
-     The message tracer feature for WSO2 DAS is no longer supported and removed from the product.
-
-- **Hazelcast Based Gateway Clustering**
-
-     The Hazelcast based gateway clustering is no longer supported and removed from the product.
+??? note "Introducing MI control plane, Search and Pagination"
```

Comment:
This is incorrect I think. This change involves the removal of a database to handle backend stuff in the dashboard plus an improved search and pagination for MI dashboard and the Management API. Nothing there about a new Control Plane as far as I know.

Reply:
@tharikaGitHub Please confirm the comment about this feature. 

Thanks

Reply:
We can change the title as "MI Dashboard Revamp and Pagination and Search capability for MI Management API"